BUG: compile fail on Linux and Macos

After merging commit to b842fd1d379b19ff5be2f0b6a1ec8c7244b98f0d, compile error occurs on Linux and Macos platform (same code compile success on Windows):

Linux:

```
 ../gframe/game.cpp: In member function 'void ygo::Game::MainLoop()':
 ../gframe/game.cpp:1078:9: error: 'std::this_thread' has not been declared
  1078 |    std::this_thread::sleep_for(std::chrono::milliseconds(20));
       |         ^~~~~~~~~~~
 ../gframe/game.cpp:1093:7: error: 'std::this_thread' has not been declared
  1093 |  std::this_thread::sleep_for(std::chrono::milliseconds(500));
       |       ^~~~~~~~~~~
```

Macos:

```
 ../gframe/game.cpp:1078:22: error: no member named 'sleep_for' in namespace 'std::this_thread'
                         std::this_thread::sleep_for(std::chrono::milliseconds(20));
                         ~~~~~~~~~~~~~~~~~~^
 ../gframe/game.cpp:1093:20: error: no member named 'sleep_for' in namespace 'std::this_thread'
         std::this_thread::sleep_for(std::chrono::milliseconds(500));
         ~~~~~~~~~~~~~~~~~~^
 2 errors generated.
```

--- a/gframe/game.cpp
+++ b/gframe/game.cpp
@@ -10,6 +10,7 @@
 #include "duelclient.h"
 #include "netserver.h"
 #include "single_mode.h"
+#include <thread>
 
 const unsigned short PRO_VERSION = 0x1361;
 

